kupyna/README.md

```diff
@@ -0,0 +1,55 @@
+# RustCrypto: Kupyna
+
+
+![Apache2/MIT licensed][license-image]
+![Rust Version][rustc-image]
+[![Project Chat][chat-image]][chat-link]
+
+Pure Rust implementation of the [Kupyna] cryptographic hash function defined under DSTU 7564:2014.
+## Examples
+
+```rust
+use kupyna::{Digest, Kupyna256};
+
+let mut hasher = Kupyna256::default();
+hasher.update(b"my message");
+let hash = hasher.finalize();
+
+println!("Computed Hash: {:02X?}", hash);
+```
+
+Also, see the [examples section] in the RustCrypto/hashes readme.
+
+## Minimum Supported Rust Version
```

Comment:
After MSRV bump to 1.85 you can remove this section since we will rely on the MSRV-aware resolver.